### PR TITLE
GSCloud 1.7-RC. Updated documentation. Pgconfig+ Geoserver Acl integration. Working example.

### PR DESCRIPTION
**Pull request by jemacchi**
- GSCloud 1.7-RC2. 
- Updated documentation. 
- Pgconfig+Geoserver Acl integration. 
- Working example

### CODE REVIEW

File: examples/pgconfig/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description: Example for geoserver-cloud with pgconfig
+name: gs-cloud-pgconfig
+version: 0.1.0
+dependencies:
+  - name: geoservercloud
+    repository: file://../../
+    version: 0.1.0
+  # Postgres dependency used for pgconfig/catalog database
+  - name: 'postgresql'
+    version: 14.0.0
```
Comment (groldan) [inline examples/pgconfig/Chart.yaml:12]: `pgconfig` requires postgres 15+

File: examples/pgconfig/templates/configmap.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+metadata:
+    creationTimestamp: null
+    name: geoserver-datasources-env
+data:
+    values.yml: |-
+        backend.pgconfig: true
+        pgconfig.jndi-name: java:comp/env/jdbc/pgconfig
+        pgconfig.schema: public
+        gwc.enabled: false
+        logging.level.org.springframework.retry.support: info
```
Comment (groldan) [inline examples/pgconfig/templates/configmap.yml:12]: * `backend.pgconfig: true` is already set by the `pgconfig` spring profile in 
```
SPRING_PROFILES_ACTIVE:
    value: standalone,pgconfig
```
* I would change the default schema from `public` to `pgconfig` or so. Flyway will create it if it doesn't exist. (also change it in `jndi.datasources.pgconfig.schema` below.

* No need to disable gwc nor set that logging level

**Review by groldan — CHANGES_REQUESTED** [review]

File: examples/pgconfig/Chart.yaml (hunk 1 above)
Comment (jemacchi) [reply examples/pgconfig/Chart.yaml:12]: It's the bitnami helmchart release, not the Postgres one. Release 14 includes Postgres 16.x.
https://github.com/bitnami/charts/tree/main/bitnami/postgresql#upgrading

File: values.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ podSecurityContext: {}
 securityContext: {}
 
 common-image-stuff: &common-image-stuff
-  tag: '1.5.2'
+  tag: '1.7-RC'
```
Comment (danduk82) [inline values.yaml:6]: it would be good to have a true release of 1.7 before merging this PR

**Review by danduk82 — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 3 above)
Comment (jemacchi) [reply values.yaml:6]: Moved to 1.7-RC since we have people interested to test the RC.  @groldan any comment/thoughts?

Comment (groldan) [reply values.yaml:6]: @danduk82 the point of an RC is to be battle tested and either become the final release or fix some urgent bug and then release. I don't  know how do you handle that in the helm chart, but to that effect we need a helm chart release.
Chances are, create a 1.7 branch and merge there, then tag it so interested parties can test it?
On another note, wouldn't it be easier to follow if the helm chart version followed the gscloud version?

File: examples/pgconfig/templates/configmap.yml (hunk 2 above)
Comment (jemacchi) [reply examples/pgconfig/templates/configmap.yml]: done

**Review by groldan — APPROVED** [review]
LGTM

**Pull request merged**